src/DataViews/SamplingReportDataView.cpp

```diff
@@ -340,6 +339,34 @@ void SamplingReportDataView::SetStackEventsCount(uint32_t stack_events_count) {
   stack_events_count_ = stack_events_count;
 }
 
+std::string SamplingReportDataView::GetToolTip(int row, int column) {
+  if (column != kColumnInclusive && column != kColumnExclusive) {
+    return "";
+  }
+  const SampledFunction& function = GetSampledFunction(row);
+  uint32_t raw_count = {};
+  float percentage = {};
+  if (column == kColumnInclusive) {
+    raw_count = function.inclusive;
+    percentage = function.inclusive_percent;
+  } else {
+    raw_count = function.exclusive;
+    percentage = function.exclusive_percent;
+  }
+
+  orbit_statistics::BinomialConfidenceInterval interval =
+      app_->GetConfidenceIntervalEstimator().Estimate(percentage / 100.0f, stack_events_count_);
+
+  return absl::StrFormat(
+      "The function %s\n"
+      "has been encountered %u times in a total of %u stack samples.\n"
+      "This makes up for %.2f%% of samples.\n"
+      "The 95%% confidence interval for the true percentage is\n"
```

Comment:
I would consider an empty line in between these two for better readability but again I don't know, I would have to see it.

Reply:
To answer your question: I like it too.